hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveMetastore.java

```diff
@@ -273,13 +274,13 @@ private void initConf(HiveConf conf, int port) {
   }
 
   private static void setupMetastoreDB(String dbURL) throws SQLException, IOException {
-    Connection connection = DriverManager.getConnection(dbURL);
-    ScriptRunner scriptRunner = new ScriptRunner(connection, true, true);
+    try (Connection connection = DriverManager.getConnection(dbURL)) {
```

Comment:
I think we should still fail if this doesn't work, so Try with resources is good but we should rethrow after closing

Reply:
NVM I didn't read the standard enough. Looks like we should still rethrow